### ISSUE TITLE
1.6.13 Mystical Agriculture errors

### ISSUE THREAD

**Issue by Ankrhim** [AllTheMods/ATM-6#1923]
Version 1.6.13 - Cannot create seeds with the infusion altar and can't remove planted seeds from (Hopper) Botany Pots.

**To Reproduce**
Have tried several combinations on the infusion altars (wood, gold, stone etc.) but nothing gets consumed and therefor the seeds are not created.

Seeds already planted in (Insanium) soil inside Botany Pots can't be manually removed, Pot needs to be broken before able to rebuild/replant. 

**Additional context**
I've created a new ATM profile in CurseForge an opened my save in the new profile to see if that fixes it, but the problem is persistent.

**Comment by BovineShaman** [comment]
**For creating seeds:** Did you give it the correct essence to go with the material? Did you provide redstone power and the correct seed to the middle altar?
**For getting seeds out of botany pots:** Try crouch right-clicking the pot with an empty hand.

**Comment by Ankrhim** [comment]
Yes, I did give the correct essense, materials and seeds. Also redstone is power is added. I just tried a new world real quick (creative) and the problem isn't there. It appears to be in my save, like something is corrupted. Because next to this problem, crouch right-clicking isn't working and wands aren't functioning in my save. 

I haven't tested these yet in the new world, but I'll guess that will also work there.
